example/lib/main.dart

```diff
@@ -37,17 +37,19 @@ class _ChewieDemoState extends State<ChewieDemo> {
   void dispose() {
     _videoPlayerController1.dispose();
     _videoPlayerController2.dispose();
-    _chewieController.dispose();
+    _chewieController?.dispose();
```

Comment:
This change is not strictly related to the migration, but it makes the test pass.